### ISSUE TITLE
[QUESTION] Is `containerfiles` attribute supposed to be paths in relative to the path declared in `context` attribute?

### ISSUE THREAD

**Issue by runlevel5** [redhat-actions/buildah-build#134]
### Question

Well I have a Containerfile which resides in a subfolder `./my_app`, for docker-compose spec file I simply declare the `context: ./my_app` and `dockerfile: Containerfile`. However for buildah-build action, I have to declare the full path for `containerfiles: ./my_app/Containerfile`. That behaviour caught me by surprise... so wondering if such decision is deliberate or simply it is just an oversight from the onset. Many thanks

**Comment by lehmanju** [comment]
Can confirm, it's unintuitive in my opinion.

**Comment by antony-jr** [comment]
Yep, wasted 10 mins on this, this should be documented somewhere, I see no proper examples with context.

**Comment by fredericlebel** [comment]
Me too, until I found this issue.

**Comment by snecklifter** [comment]
The `containerfiles` paths are resolved relative to the **workspace root** (i.e., `$GITHUB_WORKSPACE`), not relative to the `context` directory. This means if your Containerfile is at `./my_app/Containerfile`, you need to specify the full path:

```yaml
- uses: redhat-actions/buildah-build@v2
  with:
    context: ./my_app
    containerfiles: ./my_app/Containerfile
```

This differs from Docker Compose, where `dockerfile` is relative to `context`. The behaviour is tracked as a known issue in #44, which remains open as a potential improvement.

As a workaround, you can always use the full path from the workspace root for both `context` and `containerfiles`.
